### ISSUE TITLE
`pytest.lazy_fixture` doesn't exist in 0.4.2

### ISSUE THREAD

**Issue by youknowone** [TvoroG/pytest-lazy-fixture#30]
When I downgrade it to 0.4.1, the problem is fixed.

Examples
- Broken: https://travis-ci.org/youknowone/itunes-iap/jobs/438342600
- Fixed: https://travis-ci.org/youknowone/itunes-iap/jobs/438538498
- Patch: https://github.com/youknowone/itunes-iap/commit/ef4a927db45c4f9c9e62ce9653d8a5efff906548

**Comment by TvoroG** [comment]
Yes, `pytest_namespace` was [deprecated](https://docs.pytest.org/en/latest/deprecations.html#pytest-namespace) and we started to use `pytest_configure`, but the best way to use `pytest_lazyfixture` would be to just import it from library: `from pytest_lazyfixture import lazy_fixture`. Check out also this [issue](https://github.com/TvoroG/pytest-lazy-fixture/issues/16).

I will need to add this information to readme.

**Comment by youknowone** [comment]
I looked this page to solve the problem: https://pypi.org/project/pytest-lazy-fixture/
And it seems it is the README file. (redistribution will be required, right?)

I also think it shouldn't be `0.4.2` but `0.5.0`. After checking the docs, I also checked the version. And I didn't expect compatibility problem when I saw the last version was `0.4.1`

**Comment by TvoroG** [comment]
Yeah, you are right, I've messed up version :) Next time need to be more careful

**Comment by jayvdb** [comment]
Duplicate of https://github.com/TvoroG/pytest-lazy-fixture/issues/22 ?

**Comment by TvoroG** [comment]
I think #22 is a combination of this issue and #24
